.zenodo.json

```diff
@@ -50,14 +50,19 @@
         },
         {
             "orcid": "0000-0002-4437-0770",
-            "affiliation": "University of Manchester",
```

Comment:
Until tomorrow(!) my affiliation should be "University of Oxford", and I think because the majority of the work was done whilst there, that is what it should be for the JOSS paper. I would also then add "IH acknowledges support from the Beecroft Trust." to the acknowledgements.

Reply:
If multiple affiliations are possible, then I think "University of Oxford, University of Manchester" would be correct for the JOSS paper, then I will change it to Cardiff afterwards.

Reply:
I am unclear whether it is possible to have multiple affiliations. Any idea anyone?

Reply:
Agreed, we can then update to Cardiff in a separate PR in time for the next release (v0.5)

Reply:
In the JOSS paper manuscript, you have both affiliations listed. This update is just to zenodo (and in particular the v0.4.1 release on zenodo that will accompany the JOSS paper). I don't think multiple affiliations is possible through zenodo so just your lead affiliation (Oxford) is probably suitable here.

Reply:
To be clear, if you go to the zenodo record for our previous releases (e.g. [HERE](https://zenodo.org/record/4475347)) your "affiliation" is just the text that appears when you hover the cursor over your name. Nothing fancier than that as far as I can tell.